building-apk.md

```diff
@@ -4,17 +4,17 @@ title: Building an APK for Android
 permalink: /building-apk/
 ---
 
-_Note: An `apk/AndroidManifest.xml` file was already created for
```

Comment:
mentioning the manifest file seems useful, don't we want to mention this somewhere?